src/model/stationDAO.js

```diff
@@ -0,0 +1,35 @@
+import LineDAO from "./lineDAO.js";
+
+export default class StationDAO {
+  constructor() {
+    this.localStorage = window.localStorage;
+  }
+  getAllStations() {
+    if (!this.localStorage.getItem("stations")) return [];
+    return this.localStorage.getItem("stations").split(",");
+  }
+  addStation(stationName) {
+    let stations = this.getAllStations();
+    if (stations.indexOf(stationName) != -1) return null;
+    stations.push(stationName);
+    this.localStorage.setItem("stations", stations.toString());
```

Comment:
```suggestion
    this.localStorage.setItem("stations", String(stations));
```
다음과 같은 경우 stations를 이 함수내에서 보장할 수 없어서 sideEffect가 조금 더 적은 String을 사용하는게 좋을것 같아 리뷰했습니다.

```js
var value = null;
String(null);     // "null"
value.toString(); // TypeError
```

실제로 String 함수로 호출되는 생성자는 `value + '';`와 같습니다.

https://stackoverflow.com/questions/3945202/whats-the-difference-between-stringvalue-vs-value-tostring